long-running/README

```diff
@@ -0,0 +1,11 @@
+In order to load the long running model data you need access to the apache
+logs for Jujucharm.com and there are two servers there.
+
```

Comment:
I find this README a bit confusing as it is in the `longrunning` subdirectory but the commands specified need to be run in the parent as there is no Makefile here.

It may be less confusing to just include anything you need to say about `longrunning` in the parent README.